### DISCUSSION TITLE
Some questions for graph classification

### DISCUSSION THREAD

**Discussion by MNLei** [pyg-team/pytorch_geometric#2416] [Q&A]
Each graph has a different number of nodes, so each graph has a different size of X. So how do I use batch_size in DataLoder

**Comment by rusty1s** [comment]
This is described [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/introduction.html#mini-batches) and [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/batching.html). Hope this helps!
